src/cheri_insts.sail

```diff
@@ -133,8 +134,12 @@ union clause ast = CJALR : (bits(12), regidx, regidx)
  *
  * An exception is raised if:
  *   - *cs1*.**tag** is not set.
- *   - *cs1* is sealed and is not a sentry.
- *   - *cs1* is a sentry and *imm* $\ne$ 0.
+ *   - *cs1* is sealed and any of the following holds:
+ *     (a) *imm* $\ne$ 0
+ *     (b) *cd* = *cnull*, *cs1* = *cra* and not backward sentry
+ *     (c) *cd* = *cnull*, *cs1* $\ne$ *cra* and not forward inheriting sentry
+ *     (d) *cd* $\ne$ *cnull* and backward sentry
+ *     (e) *cd* $\ne$ *cra* and (forward enabled or forward disabled sentry)
```

Comment:
I think we should distinguish register index from contents here. Also useful to indicate what each case represents.
```suggestion
 *   - *cs1* is sealed and any of the following holds:
 *     (a) *imm* $\ne$ 0
 *     (b) *cd* = *cnull*, *cs1* = *cra* and *cs1* does not contain backward sentry (function return aka `cret` is only permitted to backwards sentry)
 *     (c) *cd* = *cnull*, *cs1* $\ne$ *cra* and *cs1* does not contain a forward inheriting sentry (tail calls are only permitted to interrupt inheriting sentries)
 *     (d) *cd* $\ne$ *cnull* and *cs1* contains a backward sentry (calls are only permitted to forward sentries)
 *     (e) *cd* $\ne$ *cra* and *cs1* is a forward enabled or forward disabled sentry (calls that might change interrupt status must use *cra* as link register)
```